src/pages/dashboard/Dashboard.tsx

```diff
@@ -21,12 +24,39 @@ const StyledOtherContent = styled.div`
   grid-area: search-bar;
 `;
 
-const Dashboard: React.FC = () => (
-  <StyledDashboard>
-    <StyledSearchBar resetSearchInput />
+const StyledSearchBarContainer = styled.div`
+  width: 35rem;
+`;
+
+const Dashboard: FC = () => {
+  const history = useHistory();
+  const dispatch = useDispatch();
+  const [searchTerm, setSearchTerm] = useState('');
+
+  const handleSearch = async () => {
+    if (searchTerm.length) {
+      await search(searchTerm, dispatch);
```

Comment:
vi hadde vel en diskusjon om search results egentlig hører til i redux for noen uker siden, og fant ut at det kanskje ikke trengs der. Men det er uansett en annen oppgave

Reply:
ja, har ikke gjort noe med søkefunksjonalitet, bare fått det til å funke slik som det var